src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -324,6 +314,29 @@ class ReviewCollapsibleChapter extends React.Component {
     );
   };
 
+  focusOnPage = key => {
+    const name = `${key.replace(/:/g, '\\:')}`;
```

Comment:
Some day I'll be a regex expert. Can you leave a comment explaining what this regex does, and what the value of `key` is in this instance? Thanks!

Reply:
I actually don't know what this regex does! I just moved this function to prevent a linter warning.